### PR TITLE
fix: cross platform global

### PR DESCRIPTION
**Pull request by fireridlle**
Original discussion https://github.com/fauna/faunadb-js/pull/428

Cross-platform global object (globalThis, window, self, global)

### CODE REVIEW

**Review by tigger9flow — DISMISSED** [review]

File: src/_util.js [hunk 1]
```diff
@@ -451,7 +451,17 @@ function mergeObjects(obj1, obj2) {
   return obj3
 }
 
+var crossGlobal =
```
Comment (erickpintor) [inline src/_util.js:454]: Could you maybe add some docs here explaining the motivation for this? Abdicating from using global `window` variable might be counter intuitive without further context.

Comment (fireridlle) [reply src/_util.js:454]: almost all platforms supports globalThis, which is kinda built-in polyfill for global context to cover cross platforms. but we can't use it as IE11 doesn't support it https://caniuse.com/?search=globalThis

window - browser global context
global - nodejs global context
self - service worker global context

so crossGlobal will contain one of them depends on availability

Comment (erickpintor) [reply src/_util.js:454]: That's good information, would you mind adding that as a comment to the code?

**Review by parkhomenko — DISMISSED** [review]

**Review by erickpintor — DISMISSED** [review]
🐑 it.

**Review by tigger9flow — DISMISSED** [review]

**Review by parkhomenko — APPROVED** [review]

**Review by parkhomenko — APPROVED** [review]

**Review by tigger9flow — APPROVED** [review]

**Pull request merged**